cvat-canvas/src/typescript/drawHandler.ts

```diff
@@ -491,9 +531,15 @@ export class DrawHandlerImpl implements DrawHandler {
                 }
             }
         } else if (this.drawData.shapeType === 'rectangle') {
-            this.drawBox();
-            // Draw instance was initialized after drawBox();
-            this.shapeSizeElement = displayShapeSize(this.canvas, this.text);
+            if (this.drawData.boxDrawingType === 'by_two_points') {
```

Comment:
Actually boxDrawingType declared as optional. So, it would be better to redesign this condition like:
```
this.drawData.boxDrawingType === 'by_four_points' =>> draw with four points
else =>> draw with two points
```

Reply:
Thanks, fixed it [here](https://github.com/opencv/cvat/pull/1127/commits/c400fb04d52c1baa4867753bcad6e5da360bd4a0#diff-1286785df8910c7d3f05a91f3a96e17dR582)